### ISSUE TITLE
Update Type if an entity already exists

### ISSUE THREAD

**Issue by Sgidwani61** [wjohnson/pyapacheatlas#259]
I'm trying to use the parse_bulk_entities to update all my entities in my purview data catalog in the format 

typeName, qualifiedName, name, description, type

everything updates as expected except the type column. For example I have extracted all my data from purview into excels. If the type is exported as a string but i want to update it to an int, i am changing the value in the excel being passed into the upload_entities column from string to int but that change deos not reflect in the purview data catalog. Is this expected? Is there any way to update data types for entities that already exist?

**Comment by wjohnson** [comment]
HI, @Sgidwani61 - Thank you for using PyApacheAtlas!

Do you know the typeName of the asset you're working with?  For example, sometimes the Purivew UI is tricky and the attributeName is actually something like `data_type` but the UI shows `Type`.  If you can share the asset type you can look up all of the attribute names using the [get_all_typedefs](https://wjohnson.github.io/pyapacheatlas-docs/latest/core/client/api/pyapacheatlas.core.client.PurviewClient.get_all_typedefs.html) method.

**Comment by Sgidwani61** [comment]
Yeah, im able to extract the type correctly using the data_type or type values to extract the data but what i mean is when using the bulk upload method, if i want to change a type of an already existing column for example, from string to int, i am unable to do so. Im wondering if its because there is no type value being stored anywhere in that method.
